hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -165,10 +166,17 @@ def check_flags(self, options, required_flags):
                 LOG.error(msg)
                 raise exception.InvalidInput(reason=msg)
 
-    def _create_client(self):
-        cl = client.HPE3ParClient(
-            self.config.hpe3par_api_url,
-            suppress_ssl_warnings=CONF.suppress_requests_ssl_warnings)
+    def _create_client(self, timeout=None):
+        try:
+            cl = client.HPE3ParClient(
+                self.config.hpe3par_api_url, timeout=timeout,
+                suppress_ssl_warnings=CONF.suppress_requests_ssl_warnings)
+        except Exception as ex:
+            msg = (_("Failed to connect to %(url)s, please check url") %
+                   {'url': self.config.hpe3par_api_url})
```

Comment:
Please log a message as below

Failed to connect to the array using <<URL_goes_here>>. Please ensure the following:
1. Value of IP and port specified for hpe3par_api_url in hpe.conf is correct and
2. The array is reachable from the host